src/assets/styles/app.css

```diff
@@ -83,3 +67,57 @@ article .content + .content {
 .bg-green-800 {
     color: var(--fl-fgColor, var(--color-white));
 }
+
+.footer {
+    padding-block: var(--space-xl) var(--space-3xl);
+}
+
+.footer .logo {
+    align-items: center;
+    display: flex;
+    gap: var(--space-2xs);
+    inline-size: 100%;
+    margin-block: var(--space-xl);
+}
+
+.footer .logo svg {
+    inline-size: calc(var(--space-3xs) + var(--space-xl));
+}
+
+.footer-content {
+    display: grid;
+    gap: var(--space-2xl);
+    grid-template-columns: repeat(3, 1fr);
+}
+
+.footer-credit,
+.footer-resources,
+.footer-sponsors {
+    display: flex;
+    flex-direction: column;
+    gap: var(--space-xs);
+}
```

Comment:
I'm wondering if you should give these a more generic class, e.g. `.footer-column`. If there's no difference between them.